core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -351,6 +351,17 @@ workflows:
         tag: latest
   cucushift-installer-rehearse-azure-ipi:
     platform: azure
+  cucushift-installer-rehearse-azure-ipi-ovn-hypershift-guest:
+    platform: azure-qe
+    base_images:
+      hypershift-operator:
+        name: "4.16"
+        namespace: ocp
+        tag: hypershift-operator
+      upi-installer:
```

Comment:
Does `azure-ipi-ovn-hypershift-guest` require `upi-installer` ?

Reply:
Yep. `cucushift-hypershift-extended-enable-qe-catalogsource` and `cucushift-hypershift-extended-enable-qe-pull-secret` require upi-installer.

Reply:
Ack